### PR TITLE
add a sql question to quiz users's knowledge on quering dates selections

### PR DESCRIPTION
**Pull request by ht-l1**
## Summary of changes

add a sql question to quiz users's knowledge on quering dates selections

## Checklist

- [x] I have read and followed the [contribution guidelines](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CONTRIBUTING.md).
- [x] I have read through the [Code of Conduct](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CODE_OF_CONDUCT.md) and agree to abide by the rules.
- [x] This PR is for one of the available issues and is not a PR for an issue already assigned to someone else.
- [x] My PR title has a short descriptive name so the maintainers can get an idea of what the PR is about.
- [x] I have provided a summary of my changes.

closes #XXXXX

### CODE REVIEW

File: src/data/sql-quiz.ts [hunk 1]
```diff
@@ -1020,6 +1020,19 @@ const sqlQuiz = [
     Explanation:
       "The GROUP_CONCAT function is used to concatenate strings from multiple rows into a single row. This is useful when you need to combine multiple values into a single string.",
     Link: "https://www.freecodecamp.org/news/sql-tips-save-time-write-simpler-queries/"
+  },
+  {
+    Question:
+      "Which of the following query records within a 7-day period ending on 10/15/2023?",
```
Comment (jdwilkin4) [inline src/data/sql-quiz.ts:1026]:
```suggestion
      "Which of the following SQL queries correctly shows the records within a 7-day period ending on 10/15/2023?",
```

File: src/data/sql-quiz.ts [hunk 2]
```diff
@@ -1020,6 +1020,19 @@ const sqlQuiz = [
...
+    Answer: "WHERE date >= '2023-10-09' AND date <= '2023-10-15'",
+    Distractor1:
+      "WHERE activity_date > '2023-10-15' AND activity_date <= '2023-10-08'",
+    Distractor2:
+      "WHERE activity_date >= '10/09/2023' AND activity_date <= '10/15/2023'",
+    Distractor3: "WHERE activity_date BETWEEN '2023-10-09' AND '2023-10-16'",
+    Explanation:
+      "To select some particular date between one date and another, you can use the `BETWEEN` and `AND` keywords while specifying the dates. Or simply using comparison operators such as greater than >, less than <, or euqal=.",
```
Comment (jdwilkin4) [inline src/data/sql-quiz.ts:1034]: removing backticks

```suggestion
      "To select some particular date between one date and another, you can use the BETWEEN and AND keywords while specifying the dates. You can also use comparison operators such as greater than (>), less than (<), or equal(=).",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by ht-l1** [comment]
just implemented your changes, thank you :)

**Review by jdwilkin4 — APPROVED** [review]
Congrats on another contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**